### DISCUSSION TITLE
Thanks for the hard work with the renderer

### DISCUSSION THREAD

**Discussion by davidpnewton** [Helium314/SCEE#627] [General]
I'd like to take the chance to thank all of those who have worked so hard to replace tangram-es with MapLibre.

At least for me the transition has been smooth and good. The alphas have behaved themselves and MapLibre is a definite improvement. If nothing else the opportunity to zoom in so much further allows sorting out a good few more quests for really close nodes much more easily.

Let's hope beta and final version release go as well.

**Comment by Helium314** [comment]
I'm quite curious how MapLibre is so much smoother as reported by several people, when for myself MapLibre on my Galaxy A3 2017 has noticeably worse performance than Tangram on S4 Mini. (may depend on settings, zoom, quest density, ...)

**Reply by HolgerJeromin** [reply]
We have newer phones and are so happy to be able to zoom in much futher?
It is and was fast enough for me :-)

**Comment by rusty-snake** [comment]
WOW

So much smoother. Even with my huge custom overlays.
